best-practices/workflow.md

```diff
@@ -0,0 +1,40 @@
+# Abtion's development workflow
+
+Abtion’s development workflow may vary depending on the project. Teams are encouraged to experiment with the most efficient way to reliably deliver value to our clients. The standard flow is described here. It should be the base from where teams start and the destination, when a team discovers a more efficient way of working.
+
+## Our philosophy:
+
+- We trust our colleges as professional developers.
+- We engage in discussions with an eye on productivity. We accept that a discussion might be counter-productive if the time we spend debating surpasses the benefits of the discussion.
+- We accept that we might be wrong.
+- We want to improve any codebase that we work on.
+- We want to learn.
+- We want to educate our coworkers.
+
+## Goals
+
+- Delivery quality code at a fast pace.
+- Maintain all team members informed of the changes.
+- Have a low defect rate.
```

Comment:
```suggestion
- Have a low regression rate.
```

Reply:
As I understand regression, it means working features stops working. But it doesn't not cover releasing new features defect free. Is that correct?

Reply:
Yes, that is correct, and we can certainly include both here :) Automated testing, pairing, and close client involvement should leave our defect rates at close to 0 also!